src/main/java/org/broadinstitute/hellbender/tools/exome/allelefraction/AllelicPanelOfNormals.java

```diff
@@ -0,0 +1,214 @@
+     */
+    public AllelicPanelOfNormals(final File inputFile) {
+        this(new AllelicCountCollection(validateFile(inputFile)));
+    }
+
+    private static File validateFile(final File inputFile) {
+        Utils.nonNull(inputFile);
+        Utils.regularReadableUserFile(inputFile);
+        return inputFile;
+    }
+
+    /**
+     * Gets the reference-bias alpha hyperparameter at a given SNP site if it is in the panel of normals
+     * and the MLE alpha hyperparameter across all sites if it is not.
+     * @param site  SNP site
+     * @return      reference-bias alpha hyperparameter if site is in panel of normals,
+     *              MLE alpha hyperparameter across all sites if it is not
+     */
+    public double getAlpha(final SimpleInterval site) {
+        throwExceptionIfPoNIsEmpty();
+        Utils.nonNull(site);
+        return siteToHyperparameterPairMap.getOrDefault(site, mleHyperparameterValues).alpha;
+    }
+
+    /**
+     * Gets the reference-bias beta hyperparameter at a given SNP site if it is in the panel of normals
+     * and the MLE beta hyperparameter across all sites if it is not.
+     * @param site  SNP site
+     * @return      reference-bias beta hyperparameter if site is in panel of normals,
+     *              MLE beta hyperparameter across all sites if it is not
+     */
+    public double getBeta(final SimpleInterval site) {
+        throwExceptionIfPoNIsEmpty();
+        Utils.nonNull(site);
+        return siteToHyperparameterPairMap.getOrDefault(site, mleHyperparameterValues).beta;
+    }
+
+    /**
+     * Gets the MLE mean-bias hyperparameter across all sites.
+     * @return  MLE mean-bias hyperparameter across all sites
+     */
+    public double getMLEMeanBias() {
+        throwExceptionIfPoNIsEmpty();
+        return mleMeanBias;
+    }
+
+    /**
+     * Gets the MLE bias-variance hyperparameter across all sites.
+     * @return  MLE bias-variance hyperparameter across all sites
+     */
+    public double getMLEBiasVariance() {
+        throwExceptionIfPoNIsEmpty();
+        return mleBiasVariance;
+    }
+
+    private class HyperparameterValues {
+        private final double alpha;
+        private final double beta;
+
+        private HyperparameterValues(final double alpha, final double beta) {
+            this.alpha = alpha;
+            this.beta = beta;
+        }
+
+        /**
+         * Initializes the hyperparameter values at a site given the observed counts in all normals.
+         * @param a     total alt counts observed across all normals at site
+         * @param r     total ref counts observed across all normals at site
+         */
+        private HyperparameterValues(final int a, final int r) {
+            final double f = 0.5;
+            final int n = a + r;
+            final double alpha = mleHyperparameterValues.alpha;
+            final double beta = mleHyperparameterValues.beta;
```

Comment:
See comments on earlier commit in `AlleleFractionLikelihoods` for thoughts on more expressive names for these.